cypress/integration/publicPaths/render/homepage/index.spec.ts

```diff
@@ -9,12 +13,91 @@ describe(`${pageName} renders expected components on different devices`, () => {
   const tests = [
     newExpectation(
       "renders container",
-      "[data-cy=homepage-container]",
+      "[data-cy=top-container]",
       newShouldArgs(
         "be.visible.and.contain",
         "The easiest way to build with Dolt.",
       ),
     ),
+    newExpectation(
+      "Should find top button",
```

Comment:
Nit, but should be consistent with capitalization (should vs Should)

And better descriptions for this and the one below. "should find launch database button" and "should find pricing link" or something